mm2src/lp_ordermatch.rs

```diff
@@ -4351,6 +4411,20 @@ pub async fn create_maker_order(ctx: &MmArc, req: SetPriceReq) -> Result<MakerOr
         let mut my_maker_orders = ordermatch_ctx.my_maker_orders.lock();
         my_maker_orders.insert(new_order.uuid, Arc::new(AsyncMutex::new(new_order.clone())));
     }
+
+    let balance = match base_coin.my_spendable_balance().compat().await {
```

Comment:
By the https://github.com/KomodoPlatform/atomicDEX-API/pull/1279#discussion_r867834849 comment, I meant that balance is already requested during order creation in `get_max_volume` or `check_balance_for_maker_swap`. You can refactor them to return the balance too.